backends/plugins/win32/win32-provider.cpp

```diff
@@ -36,7 +36,7 @@
 class Win32Plugin : public DynamicPlugin {
 private:
 	static const TCHAR* toUnicode(const char *x) {
-	#ifndef _WIN32_WCE
+	#ifndef UNICODE
```

Comment:
(Same comment, in a review)
Why has this been changed? The former check for _WIN32_WCE is done in other places in the code

Reply:
This is done to match a similar function in [backends/fs/windows/windows-fs.cpp](https://github.com/scummvm/scummvm/blob/master/backends/fs/windows/windows-fs.cpp#L99).